### ISSUE TITLE
Update for getCodeIndex() issue with GUI Builder for Processing (#545)

### ISSUE THREAD

**Issue by quarks** [benfry/processing4#596]
Issue 545 has been closed and locked so I have created this one.

I have released a new version of GUI Builder that no longer uses the method

`getCodeIndex(SketchCode who)`

so has resolved the original issue.

Sorry for any inconvenience 😄

**Comment by benfry** [comment]
Hm, wonder if it's safe to remove again, or if anyone else is using it in a library? Any ideas?
https://github.com/processing/processing4/commit/b934c4de965d7d8603356608daca1d7fe94b89cf

**Comment by quarks** [comment]
GUI Builder creates a second tab for the GUI creation code and this tab is read and updated as the user switches between the sketch window and the GUI design window. To access the different tabs I needed to delve into the Sketch class source code to find the methods I needed and picked `getCodeIndex()`. All this was back in 2011 and it seems I picked a method that is not called by any other Processing code LOL.
 
GUI Builder now uses the method`getCode()` method, although undocumented I suspect Processing uses it otherwise it also would have been commented out 😄 
 
I doubt that any other library or tool uses `getCodeIndex()` otherwise you would have heard either here or on the forum.
